### ISSUE TITLE
Unable to start cgw-web

### ISSUE THREAD

**Issue by lance-web3** [safe-global/safe-client-gateway#1682]
### Description

How to set the env when we are not able to start the container? 
Stop using the public docker image and pull from this repo instead?

not even under 
https://github.com/safe-global/safe-infrastructure/blob/main/container_env_files/cgw.env

### Reproduction steps

root@containers:/opt/safe-infrastructure# docker compose up -d cgw-web
```
[+] Running 2/2
 ✔ Container safe-infrastructure-cgw-redis-1  Healthy                                                                                                                                                         0.6s
 ✔ Container safe-infrastructure-cgw-web-1    Started                                                                                                                                                         0.8s
root@containers:/opt/safe-infrastructure# docker logs -f safe-infrastructure-cgw-web-1
/dist/src/config/configuration.validator.js:19
    throw Error(`Configuration is invalid: ${errors}`);
          ^

Error: Configuration is invalid: INFURA_API_KEY Required; RELAY_PROVIDER_API_KEY_ARBITRUM_ONE Required
    at default_1 (/dist/src/config/configuration.validator.js:19:11)
    at Object.validate (/dist/src/config/configuration.module.js:27:70)
    at ConfigModule.forRoot (/node_modules/@nestjs/config/dist/config.module.js:78:45)
    at ConfigurationModule.register (/dist/src/config/configuration.module.js:25:39)
    at AppModule.register (/dist/src/app.module.js:97:60)
    at bootstrap (/dist/src/main.js:7:94)
    at Object.<anonymous> (/dist/src/main.js:12:6)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)

Node.js v20.14.0
```

docker-compose.yml
```
  cgw-web:
    image: safeglobal/safe-client-gateway-nest:${CGW_VERSION}
    env_file:
      - container_env_files/cgw.env
    depends_on:
      cgw-redis:
        condition: service_healthy
```

### Expected behavior

able to start and set the env under 
https://github.com/safe-global/safe-infrastructure/blob/main/container_env_files/cgw.env

**Comment by lance-web3** [comment]
add ourself on cgw.env lol

**Comment by hectorgomezv** [comment]
Sorry for being late, but this is fixed by https://github.com/safe-global/safe-infrastructure/pull/144

**Comment by lance-web3** [comment]
> Sorry for being late, but this is fixed by [safe-global/safe-infrastructure#144](https://github.com/safe-global/safe-infrastructure/pull/144)

okay thanks,

desperately need help on 
https://github.com/safe-global/safe-client-gateway/issues/1692

https://github.com/safe-global/safe-infrastructure/issues/143
